digits/pretrained_model/job.py

```diff
@@ -14,16 +14,16 @@ class PretrainedModelJob(Job):
     """
 
     def __init__(self, weights_path, model_def_path, labels_path=None,framework="caffe",
-                image_type="3",resize_mode="Squash", width=224, height=224, **kwargs):
+                image_type="3",resize_mode="Squash", height=224, width=224, **kwargs):
```

Comment:
I wanted to be more consistent with my ordering of height and width for input dimensions